framework/include/araui/ui_widget.h

```diff
@@ -354,6 +354,19 @@ ui_error_t ui_widget_set_userdata(ui_widget_t widget, void *userdata);
  */
 void *ui_widget_get_userdata(ui_widget_t widget);
 
+/**
+ * @brief Set the animation of the widget.
+ *
+ * Animation finished callback can be NULL.
+ *
+ * @param[in] widget Handle of the widget
+ * @param[in] anim Animation information
```

Comment:
I think `Handle of the animation` is better.

Reply:
Fixed